pkg/ds/daemon_set.go

```diff
@@ -161,8 +150,17 @@ func (ds *daemonSet) WatchDesires(
 					continue
 				}
 
-			case <-deletedCh:
-				ds.logger.NoFields().Infof("Received daemon set delete signal")
+			case deleteDS := <-deletedCh:
```

Comment:
is the caller of `WatchDesires()` going to be able to know the contents of the deleted daemon set? I would think all we'll know is the uuid of it